caravel/bin/caravel

```diff
@@ -17,6 +17,8 @@ from caravel import data, utils
 from caravel import db
 
 config = app.config
+logging.config.dictConfig(config.get('LOGGING_CONFIG'))
```

Comment:
Thats a good catch. I tested it after removing the redundant declaration and it works.